dlp/snippets/src/main/java/dlp/snippets/InspectDataToHybridJobTrigger.java

```diff
@@ -0,0 +1,119 @@
+    String textToDeIdentify = "My email is test@example.org and my name is Gary.";
+    inspectDataToHybridJobTrigger(textToDeIdentify, projectId, jobTriggerId);
+  }
+
+  // Inspects data using a hybrid job trigger.
+  // Hybrid jobs trigger allows to scan payloads of data sent from virtually any source for
+  // sensitive information and then store the findings in Google Cloud.
+  public static void inspectDataToHybridJobTrigger(
+      String textToDeIdentify, String projectId, String jobTriggerId) throws Exception {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DlpServiceClient dlpClient = DlpServiceClient.create()) {
+      // Specify the content to be inspected.
+      ContentItem contentItem = ContentItem.newBuilder().setValue(textToDeIdentify).build();
+
+      // Contains metadata to associate with the content.
+      // Refer to https://cloud.google.com/dlp/docs/reference/rest/v2/Container for specifying the paths in container object.
+      Container container =
+          Container.newBuilder()
+              .setFullPath("10.0.0.2:logs1:app1")
+              .setRelativePath("app1")
+              .setRootPath("10.0.0.2:logs1")
+              .setType("logging_sys")
+              .setVersion("1.2")
+              .build();
+
+      HybridFindingDetails hybridFindingDetails =
+          HybridFindingDetails.newBuilder().setContainerDetails(container).build();
+
+      HybridContentItem hybridContentItem =
+          HybridContentItem.newBuilder()
+              .setItem(contentItem)
+              .setFindingDetails(hybridFindingDetails)
+              .build();
+
+      // Activate the job trigger.
+      ActivateJobTriggerRequest activateJobTriggerRequest =
+          ActivateJobTriggerRequest.newBuilder()
+              .setName(JobTriggerName.of(projectId, jobTriggerId).toString())
+              .build();
+      DlpJob dlpJob = dlpClient.activateJobTrigger(activateJobTriggerRequest);
+
+      // Build the hybrid inspect request.
+      HybridInspectJobTriggerRequest request =
+          HybridInspectJobTriggerRequest.newBuilder()
+              .setName(JobTriggerName.of(projectId, jobTriggerId).toString())
+              .setHybridItem(hybridContentItem)
+              .build();
+
+      // Send the hybrid inspect request.
+      dlpClient.hybridInspectJobTrigger(request);
+
+      // Build a request to get the completed job
+      GetDlpJobRequest getDlpJobRequest = GetDlpJobRequest.newBuilder()
+              .setName(dlpJob.getName())
+              .build();
+      DlpJob result = null;
+      do {
+        result = dlpClient.getDlpJob(getDlpJobRequest);
+        Thread.sleep(5000);
+      } while (result.getInspectDetails().getResult().getProcessedBytes() <= 0);
```

Comment:
```suggestion
      dlpServiceClient.getDlpJobCallable()
          .futureCall(getDlpJobRequest)
          .get(3, TimeUnit.MINUTES);
```

Consider something like this for long-running operations.